backend/OWNERS

```diff
@@ -1,6 +1,10 @@
 approvers:
   - vicaire
   - IronPan
+  - neuromage
+  - yebrahim
```

Comment:
Approver list is for a couple folks to help maintain consistency / best practices / etc. I would like to keep the list as is for now. (See https://github.com/kubernetes/community/blob/master/community-membership.md)

Feel free to add yourself to reviewer.